library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BillableCallDoctrineRepository.php

```diff
@@ -75,7 +75,7 @@ public function findUnratedInGroup(array $pks)
 
         $conditions = [
             ['self.id', 'in', $pks],
-            ['self.invoice', 'in', $pks],
+            ['trunksCdr.cgrid', 'isNull'],
```

Comment:
We should add ['self.invoice', 'isNull', null] as well. There is no inner validation to avoid invoiced call rerating (just the one on web client).